test/mocks/DataSourceMock.scala

```diff
@@ -0,0 +1,34 @@
+package mocks
+
+import model.{Device, Position, Sensor}
+import org.mockito.Mockito.{mock, when}
```

Comment:
I don't like mockito as on real world situations it just complicates the tests code and includes possible issues on runtime